client/src/app/game-room.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GameRoomService {
+
+  public gameID: string;
+  public playerName: string;
+  public allPlayers: string[];
```

Comment:
string[] is id?
if not its need to be id + name

Reply:
its list of names